niworkflows/utils/tests/test_misc.py

```diff
@@ -39,6 +39,7 @@ def test_fs_license_check2(monkeypatch):
     assert check_valid_fs_license() is True
 
 
+@pytest.mark.skipif(not os.getenv("FREESURFER_HOME"), reason="FreeSurfer not installed")
```

Comment:
```suggestion
```
Shouldn't this test always pass (regardless of whether you have access to an fs_license or not?

The only skipif you could have is when `mri_convert` is not present in the environment

Reply:
yep, i was using `FREESURFER_HOME` as an indication of that. I guess using `shutil.which` would be more precise

Reply:
in the end I think there won't be a big difference, let's just keep the skipif.

Reply:
i did it anyways for the sake of best practices